src/api/app/assets/stylesheets/webui/timeline.scss

```diff
@@ -1,15 +1,10 @@
 // This stylesheet defines styles for use in the comments and request history merged timeline
-.timeline-item {
-    position: relative;
-}
+.timeline {
+    $timeline-border-width: 0.125rem;
+    border-left: $timeline-border-width solid $gray-400;
 
-.timeline-item:before {
```

Comment:
What this code did is render a 2px-wide gray line next to each timeline item. Instead, we can apply a left border to the timeline as a whole. This is much easier to understand.